vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -45,26 +45,17 @@ relationship OneToMany {
    Conference to Session
 }
 ```
-The above adds two fields to the Conference entity, introduces the Session entity plus an enum, and creates a mapping between the two entities. Below is the updated data model in JDL-Studio:
+This file content adds two fields to the Conference entity, introduces the Session entity plus an enum, and creates a mapping between the two entities. Below is the updated data model in JDL-Studio:
 
 ![conference.jdl](./img/jhipster-jdl.png)
 
-5. Import the JDL file into your application. You will be prompted with the option to generate MFEs if this was requested during project generation.
+4. Import the JDL file into your application. You will be prompted with the option to generate MFEs if this was requested during project generation.
 ```
 ent jhipster import-jdl conference.jdl
 ```
-If the original project structure has been retained, this step will update your data model, add entries to Liquibase to upgrade database schema during deployment, add service methods to your microservice, add fields to your MFEs, etc.
-
-6. You can now build your updated project and [run it locally](./run-local.md) or [deploy it to Entando](../pb/publish-project-bundle.md). To execute a comprehensive local test, use the following commands to build the project
-```
-ent prj build
-ent prj xk start
-ent prj be-test-run
-ent prj fe-test-run
-```
-then start Keycloak, the microservice, and one of the MFEs.
+If the default project structure has been retained, this step will update your data model, add entries to Liquibase to upgrade database schema during deployment, add service methods to your microservice, add fields to your MFEs, etc.
```

Comment:
should there be an 'and' in this kind of list?

Reply:
it's a little awkward - almost too detailed and comprehensive to end with an "etc." - but the structure avoids an "and"